src/pages/landing/stocks/_whyTradeWIthUs.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Container, Flex, SectionContainer } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import device from 'themes/device.js'
+
+const Wrapper = styled(Flex)`
+    flex-wrap: wrap;
+    flex-direction: row;
+    @media ${device.tabletL} {
+        flex-direction: column;
+        justify-content: center;
+    }
+`
+
+const ImageWrapper = styled.img`
+    margin: 0 auto 1.6rem;
+    width: 64px;
+    height: 64px;
+`
+
+const Card = styled(Flex)`
+    max-width: 38.4rem;
+    margin: 3.6rem 1.2rem;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+
+    & > img {
+        margin-bottom: 1.6rem;
+        width: 64px;
+        height: 64px;
+    }
```

Comment:
This seems to be selecting `ImageWrapper` on line 17. If ImageWrapper is being styled already, we can omit these styles.

Reply:
Removed this and use ImageWrapper instead.